ICA/auto_Granduer_ICA.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date,datetime
+
+# This funciton uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+# While loop is infinate will keep script running as long as it dosent error out.
+go=1
+while go == 1:
+    print('Looking for anaylses that need to start!')
+    #This block set the icav2 tool to correct project in use
+    bashCommand="icav2 projects enter Testing"
+    process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+    process.communicate()
+
+    #This  block collects the information of data and runs on ICA
+    bashCommand = "icav2 projectdata list --file-name UT-M03999-  --parent-folder / --match-mode FUZZY"
+    tmp= icav_out(bashCommand)
+    bashCommand = "icav2 projectdata list --file-name UT-M07101-  --parent-folder / --match-mode FUZZY"
```

Comment:
We actually have three MiSeqs (and one iSeq, but I don't think that's going to be used anytime soon)

Reply:
I should probably add those too just incase and maybe try to make it a single request to ICA.